### PR TITLE
Amit shew

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: utils/contributors.js [hunk 1]
```diff
@@ -33,6 +32,15 @@ const CONTRIBUTERS = [
...
     },
+    {
+        handle_name: 'amit-shew',
+        name: "My First Card",
+        image: "/images/amit-shew.jpg",
+        description: "Hello everyone, this is the first card by me :)",
+        github: "https://github.com/amit-shew",
+    },
+    
+
```
Comment (shubhanshu02) [inline utils/contributors.js]: Remove the extra empty line.

File: utils/contributors.js [hunk 2]
```diff
@@ -6,7 +6,6 @@ const CONTRIBUTERS = [
         description: "Hello everyone, this is the first card. Go ahead, add your own!",
         github: "https://github.com/COPS-IITBHU",
     },
-    // Start adding your data below
```
Comment (shubhanshu02) [inline utils/contributors.js:9]: Do not remove this line.

**Review by shubhanshu02 — CHANGES_REQUESTED** [review]
LGTM :+1:. Just would like to request a few changes as below.

File: utils/contributors.js (hunk 2 above)
Comment (ghost) [reply utils/contributors.js:9]: done bhaia :)
thanks a lot

**Review by shubhanshu02 — APPROVED** [review]

**Pull request closed without merging**